.github/workflows/check.yml

```diff
@@ -1,17 +1,22 @@
-name: Check
+name: Zeek tests
 
 on:
   pull_request:
   push:
     branches: [master]
+  schedule:
+    - cron: '0 0 * * *'
 
 jobs:
   pre-commit:
-    runs-on: ubuntu-20.04
-    container: zeekurity/zeek
+    # Scheduled runs don't act on new commits, so skip pre-commit analysis
+    if: github.event_name != 'schedule'
```

Comment:
The way we run this action [will run on all files](https://github.com/pre-commit/action/blob/main/action.yml#L7). While we use tags so this should find nothing interesting if hooks are not moving their tags I don't think we should go out of our way to not run this (extra benefit: less code here).

```suggestion
```